### PR TITLE
feat: Improve Form Loading Indicator (WEB-4)

### PR DESCRIPTION
**Pull request by kylemas**
ezgif com-gif-maker

### CODE REVIEW

**Comment by nandanmen** [comment]
Can you add a quick GIF of this? Also I think Najla is working on a design mockup so this might change soon. Thanks for doing it nevertheless 😄

File: components/NewsletterForm/NewsletterForm.tsx [hunk 1]
```diff
@@ -90,12 +91,18 @@ export default function NewsletterForm() {
...
+          <div className={clsx(styles.loader, 'mt-6 lg:col-span-2')}>
+            <ClipLoader size={35} color={'#00704a'} />
+          </div>
+        ) : (
+          <button
+            className={clsx(styles.submit, 'mt-6 lg:col-span-2')}
+            type="submit"
+          >
+            Submit
+          </button>
```
Comment (nandanmen) [inline components/NewsletterForm/NewsletterForm.tsx]: Could this loader potentially go in the button instead? So something like:
```suggestion
          <button
            className={clsx(styles.submit, 'mt-6 lg:col-span-2')}
            type="submit"
          >
            {state === FormState.Loading ? <ClipLoader size={35} color={'#00704a'} /> : "Submit"}
          </button>
```

I suppose this is up to the designers, but would be good to not have to duplicate the button style.

Comment (kylemas) [reply components/NewsletterForm/NewsletterForm.tsx]: Ooo okay, sure!

**Comment by kylemas** [comment]
> Can you add a quick GIF of this? Also I think Najla is working on a design mockup so this might change soon. Thanks for doing it nevertheless 😄

@narendrasss Added the gif now! Also my bad, I just saw that it was assigned to Sandy on Jira haha

**Review by nandanmen — APPROVED** [review]

**Pull request merged**